tests/postprocess_varlociraptor.nf.test

```diff
@@ -8,7 +8,7 @@ nextflow_pipeline {
     script "../main.nf"
     tag "pipeline"
     tag "pipeline_sarek"
-    profile "test,mutect"
+    profile "test"
```

Comment:
won't you need to set the mutext test profile specific things then?

Reply:
I did miss the last test using mutect2...
Put it in a separate test for now, for testing, but I'll probably revert that, as I think the actual test data needs updating instead of us patching this in the test config